lib/network/http2.go

```
@@ -130,7 +130,18 @@ func NewHTTP2Network(config *HTTP2NetworkConfig) (h2n *HTTP2Network) {
 
 // GetClient creates new keep-alive HTTP2 client
 func (t *HTTP2Network) GetClient(endpoint *common.Endpoint) NetworkClient {
-	rawClient, _ := common.NewHTTP2Client(defaultTimeout, 0, true)
+	rawClient, _ := common.NewHTTP2Client(
```

We shouldn't be ignoring errors here